homeassistant/components/npi_gpio/binary_sensor.py

```diff
@@ -0,0 +1,89 @@
+"""Support for binary sensor using NPi GPIO."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components import npi_gpio
```

Comment:
Please make all intra package imports relative.
```py
from . import setup_input, ...
```

Reply:
Done in commits 06ee5e246ec118117fac5d7c52041bfc53df0d38 6d725770b585615365e65b89f168e4b8dc5b29cf 9b427283e223f70bc1296fc7dbbb0e3b1e4cffd6 47ab26acf3610677c85f2b8da70149016f5d994f

Reply:
tested and works